dlc-manager/src/manager.rs

```diff
@@ -37,16 +37,54 @@ use std::collections::HashMap;
 use std::ops::Deref;
 use std::string::ToString;
 
-/// The number of confirmations required before moving the the confirmed state.
+/// The number of btc confirmations required before moving the DLC to the confirmed state.
```

Comment:
Why change this comment, I'm not sure "btc confirmations" makes so much sense.

Reply:
Oh I think github copilot did that, I can remove it.